test/helpers/quictestlib.c

```diff
@@ -399,19 +429,21 @@ int qtest_wait_for_timeout(SSL *s, QUIC_TSERVER *qtserv)
      */
     if (!SSL_get_event_timeout(s, &tv, &cinf))
         return 0;
-    if (using_fake_time)
-        now = fake_now;
-    else
+    if (using_fake_time) {
+        now = qtest_get_time();
+    } else {
         now = ossl_time_now();
+    }
     ctimeout = cinf ? ossl_time_infinite() : ossl_time_from_timeval(tv);
     stimeout = ossl_time_subtract(ossl_quic_tserver_get_deadline(qtserv), now);
     mintimeout = ossl_time_min(ctimeout, stimeout);
     if (ossl_time_is_infinite(mintimeout))
         return 0;
-    if (using_fake_time)
-        fake_now = ossl_time_add(now, mintimeout);
-    else
+    if (using_fake_time) {
+        qtest_add_time(ossl_time2ms(mintimeout));
+    } else {
         OSSL_sleep(ossl_time2ms(mintimeout));
+    }
```

Comment:
Same here about the curly brackets. Please see the coding style.

Reply:
The coding style does not prevent (or require) braces in this context ...

Reply:
It says:

Do not unnecessarily use braces around a single statement:

    if (condition)
        action();

and

    if (condition)
        do_this();
    else
        do_that();

Reply:
You are correct - I remembered the debates on this - not the outcome - which is to not use them around a single statement.